recipes/cython_gsl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "CythonGSL" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name }}
```

Comment:
Drop the jinja2 variable for the name (it is not that useful here), and add the `cythongsl` here and below. Conda and PyPI are case insensitive, but conda enforces lower case names.